packages/web-components/src/components.d.ts

```diff
@@ -2613,6 +2621,14 @@ declare namespace LocalJSX {
           * The href attribute of the anchor.
          */
         "href": string;
+        /**
+          * Adds an aria-label attribute to the link element.
+         */
+        "iconName"?: string;
+        /**
+          * Adds an aria-label attribute to the link element.
+         */
+        "iconSize"?: number;
```

Comment:
Do we want the icon size to be customizable? I would think that we'd want to manage that the same as the other icons.

Reply:
I understand the concern regarding icon size customization. I believe we could limit it to a maximum of two sizes for consistency and to ensure the best user experience. However, it might be more effective to provide a single fixed size to maintain uniformity across the site.

Reply:
Just keeping the icon size as is to provide flexibility.

Reply:
@harshil1793 Do you mind updating the `iconSize` prop description to say that it will accept "an integer between 3 and 9 inclusive" similar to how it's said for `va-icon`?

```
 /**
   * The size of the icon to be displayed in the link.
   */
  @Prop() iconSize?: number = 3;
```